### PR TITLE
Connect Gotts menu to Toast

### PR DESCRIPTION
**Pull request by sebko**

### CODE REVIEW

File: gotts-menu/components/Sizes.tsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ const Sizes: React.FC<Sizes> = ({ value }) => {
             key={x.id}
             sx={{
               marginRight: 3,
-              ":first-child": {
+              ":first-of-type": {
```
Comment (sebko) [inline gotts-menu/components/Sizes.tsx:19]: Fixes the follwing warning:

>The pseudo class ":first-child" is potentially unsafe when doing server-side rendering. Try changing it to "first-of-type"

File: gotts-menu/.env [hunk 2]
```diff
@@ -1,2 +1,2 @@
-RAYDIANT_MENU_API_KEY=4de9279071f84308a6815d92f6349db8
+RAYDIANT_MENU_API_KEY=dd65232a1af34ab3919b4f697587775b
```
Comment (sebko) [inline gotts-menu/.env:1]: Do we need to encrypt this with Codeship?

Comment (sebko) [reply gotts-menu/.env:1]: Figured this is exposed on the client, but no harm in encrypting in the repo - it's public after all.

File: gotts-menu/components/SubGroup2.tsx [hunk 3]
```diff
@@ -39,9 +26,6 @@ const SubGroup2: React.FC<SubGroup2> = ({ name, items }) => {
                   {name && (
                     <Text sx={{ fontStyle: "normal" }}>{name}</Text>
```
Comment (sebko) [inline gotts-menu/components/SubGroup2.tsx:27]: Now that the name contains the ABV, volume and location we can no longer italicize the `location` field, just an FYI.

File: gotts-menu/pages/menu-boards/eats-and-drinks.tsx [hunk 4]
```diff
@@ -52,7 +52,7 @@ const EatsAndDrinks: NextPage<EatsAndDrinksProps> = ({ data }) => {
 
     const id = setInterval(() => {
       refreshData();
-    }, 3000);
+    }, 30000);
```
Comment (sebko) [inline gotts-menu/pages/menu-boards/eats-and-drinks.tsx:55]: Set more sensible data refresh defaults.

File: gotts-menu/components/SubGroup2.tsx [hunk 5]
```diff
@@ -39,9 +26,6 @@ const SubGroup2: React.FC<SubGroup2> = ({ name, items }) => {
...
                   )}{" "}
-                  {location && <Text>({location})</Text>}{" "}
```
Comment (sebko) [inline gotts-menu/components/SubGroup2.tsx:42]: Dangit, doesn't look like I was italicising the "location" field. Apologies if that's why you went with merging all these text parts into the `name` field.

**Review by jschr — APPROVED** [review]

File: gotts-menu/components/SubGroup2.tsx (hunk 3 above)
Comment (jschr) [reply gotts-menu/components/SubGroup2.tsx:27]: In the old data set we we used a Location tag. We could do that again if you think it's a helpful to show an example. But if we already have good enough examples showing how to work with tags then maybe not a big deal. Up to you!

File: gotts-menu/components/SubGroup2.tsx (hunk 5 above)
Comment (jschr) [reply gotts-menu/components/SubGroup2.tsx:42]: I didn't actually set this one up. Anthony did originally. When I created the sample data I decided to go with tags so you could but totally up to you if you want to switch it.

File: gotts-menu/components/SubGroup2.tsx (hunk 3 above)
Comment (sebko) [reply gotts-menu/components/SubGroup2.tsx:27]: Looks like we do have an example of how tags work. I'll start working on the multi-location stuff today, and if that's easy, I'll come back to this.

**Pull request merged**